bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -58,6 +59,21 @@ export class AxisView extends GuideRendererView {
     // Build math_text_view if axis_label is a MathText instance
     if (axis_label != null && axis_label instanceof MathText)
       this.axis_label_math_text_view = await build_view(axis_label, {parent: this})
+
+    const {major_label_overrides} = this.model
+
+    if (major_label_overrides) {
```

Comment:
is this conditional actually necessary? The property is not nullable so the value is at least an empty object, in which case iterating over the empty object is fine (a no-op)

Reply:
It seems unnecessary, I have removed it.